packages/components/src/playground/button.spec.tsx

```diff
@@ -1,28 +1,33 @@
+import { primary, secondary } from "./button.stories";
```

Comment:
The import ordering we're enforcing here seems weird 😕

Reply:
hm yeah we're just using default https://eslint.org/docs/rules/sort-imports ... maybe should use the same rule you added to traject?

Reply:
Oh, ha, I didn't realize that eslint had a built-in rule for this.

Maybe we should use the eslint-plugin-import/order rule, and be consistent between traject, frontend-libs, axe-storybook-testing, and this. (And then of course add it to Along, too).

Reply:
yeah, that one has the `fix` I was planning on getting to that today, so no need to handle it yourself